packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -137,6 +137,14 @@ class MenuItem extends React.Component {
       { 'is-disabled': isDisabled },
     );
 
+    const ariaAttributes = {};
+    ariaAttributes['aria-haspopup'] = hasChevron;
+    ariaAttributes['aria-disabled'] = isDisabled;
+
+    if (hasChevron) {
+      ariaAttributes['aria-expanded'] = !hasChevron;
+    }
+
```

Comment:
Why do we need if condition here? If it is to check undefined / null then we should have default values declared for all rops.

Reply:
we don't want add aria-expanded for all the list items, so only when it has submenu then append this attribute. like if we don't add this condition then SR reads as collapsed for all list items even the item does not have submenu.